docs/Targets.md

```diff
@@ -3199,24 +3200,96 @@ IMAGE=test-app/image.elf SIGN=--ecc384 tools/scripts/x86_fsp/qemu/make_hd.sh
 For more advanced uses of TPM, please check [TPM.md](TPM.md) to configure wolfBoot
 according to your secure boot strategy.
 
-### Running on Kontron VX3060-S2
+## Kontron VX3060-S2
 
-A reference configuration and helper scripts are provided to run wolfBoot on
-Kontron VX3060-S2 board.
-A flash dump of the original Flash BIOS is needed.
-To compile a flashable image run the following steps:
+wolfBoot supports Kontron VX3060-S2 board using Intel Firmware Support Package
+(FSP). You can find more details about the wolfBoot support with Intel FSP in
+the above [section](#intel-x86_64-with-intel-fsp-support). A minimal
+configuration example is provided in
+[config/examples/kontron_vx3060_s2.config](config/examples/kontron_vx3060_s2.config).
+In order to produce a flashable flash image, a dump of the original flash is
+required. To build wolfBoot, follow the following steps:
```

Comment:
Just say "run the following command": now it's only one

Reply:
these are three

Reply:
OK ignore